.github/actions/run-integration-tests/action.yml

```diff
@@ -45,10 +23,11 @@ runs:
               echo "IntegrationTestsJob: Just wait until the clean-up finishes..."; \
               return; \
             fi; \
-            echo "IntegrationTestsJob: Notifying the integration tests container about the cancellation"; \
-            docker kill --signal="SIGINT" ig-integration-tests > /dev/null; \
-            echo "IntegrationTestsJob: Waiting for the integration tests container to finish"; \
-            docker wait ig-integration-tests & wait $!; \
+            echo "IntegrationTestsJob: Notifying the integration tests process about the cancellation"; \
+            PIDS=$(jobs -p)
+            kill -2 $PID > /dev/null; \
```

Comment:
Where is the PID variable initialized?

Reply:
Thanks for spotting. I was suppose to use `$PIDS` here.